src/geometry/manifold/mfderiv.lean

```diff
@@ -1227,6 +1227,82 @@ lemma mfderiv_within_const (hxs : unique_mdiff_within_at I s x) :
 
 end const
 
+section arithmetic
+/-! #### Arithmetic
+
+Note that in the in `has_mfderiv_at` lemmas there is an abuse of the defeq between `E'` and
+`tangent_space 𝓘(𝕜, E') (f z)` (similarly for `g',F',p',q'`). In general this defeq is not
+canonical, but in this case (the tangent space of a vector space) it is canonical.
+ -/
+
+variables { z : M} {F' : Type*} [normed_field F'] [normed_algebra 𝕜 F']
```

Comment:
Sorry, one more thing from me.  Does `[normed_comm_ring F']` work to replace `[normed_field F']`?

Reply:
Yep! Fixed :)